annotation_tool/docker-compose.yml

```diff
@@ -6,10 +6,12 @@ services:
       DEFAULT_ADMIN_EMAIL: "example@example.com"
       DEFAULT_ADMIN_PASSWORD: "DEMO_PASSWORD"
       NODE_ENV: "production"
-      PROD_DB_HOSTNAME: "db"
-      PROD_DB_NAME: "databasename"
-      PROD_DB_USERNAME: "somesafeuser"
-      PROD_DB_PASSWORD: "somesafepassword"
+      DB_HOSTNAME: "db"
+      DB_NAME: "databasename"
+      DB_USERNAME: "somesafeuser"
+      DB_PASSWORD: "somesafepassword"
+      COOKIE_KEYS: "somesafecookiekeys"
+      JWT_SECRET: "somesafesecret"
```

Comment:
I would comment out these two lines so we ensure nobody can deploy the tool with hardcoded secrets. We can also add a comment stating that these need to be set before `docker compose up`